spec/sidekiq-instrument/client_middleware_spec.rb

```diff
@@ -39,6 +39,39 @@
       end
     end
 
+    context 'with WorkerMetrics.enabled true' do
```

Comment:
Can we create a separate spec file for`worker_metrics` and unit tests?

Reply:
not much scope to move to another file, scenarios can be tested with same context.